ci/check-update-build-webide.sh

```diff
@@ -0,0 +1,81 @@
+#!/usr/bin/env bash
+# Copyright (c) 2019 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+set -euo pipefail
+
+GITHUB_API_TOKEN=${GITHUB_API_TOKEN:-""}
+GOOGLE_APPLICATION_CREDENTIALS_CONTENT=${GOOGLE_APPLICATION_CREDENTIALS_CONTENT:-""}
+
+#sets LATEST_SDK_WITH_TIME
+getLatestSdk() {
+  local TMP_FILE=$(mktemp)
+  local JQ_COMMAND='. | map(select(.prerelease == false)) | map([.tag_name,.published_at]) | map(.[0] + "," + .[1] + "," + (.[1]|fromdate|tostring))[]'
+  if [ -n "$GITHUB_API_TOKEN" ]; then
+    $(curl -H "Authorization: $GITHUB_API_TOKEN" https://api.github.com/repos/digital-asset/daml/releases -s > "$TMP_FILE")
```

Comment:
Note that this will fail hard if you happen to have an invalid token in `GITHUB_API_TOKEN`.